build/firmware/Common.mk

```
@@ -392,6 +393,9 @@ IS_ADMIN_CMD=net.exe session 2>&1 >/dev/null | grep -o
 verify_case_sensitivity_$(UBOOT_OUT) verify_case_sensitivity_$(OPTEE_OUT):
 ifeq (,$(findstring Microsoft,$(shell cat /proc/sys/kernel/osrelease)))
 	@echo Not running WSL, no case sensitivity check needed.
+else
+ifneq (,$(CASE_WSL_MNT))
```

```suggestion
ifneq ("",$(CASE_WSL_MNT))
```
Wasn't working for me on WSL2 without the empty string... not sure why.

---

This is really feeling like a WSL2 bug or something...

The string "" is actually two quote characters, so the ifneq will always trigger since no path is ever ""

Still trying to figure out the issue here.

---

Turns out $@ is not calculated until runtime. So it was checking based on CASE_DIR=$(subst verify_case_sensitivity_,,''), which does indeed fall through to equal the empty string.